admin/lib/validator/settings-validator.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const ValidationError = require('../validation-error')
+const errorConverter = require('../error-converter')
+const settingsErrorMessages = require('../errors/settings')
+
+const settingsValidationSchema = {
+  'questionTimeLimit': {
+    notEmpty: true,
+    isFloat: {
+      options: [{min: 1, max: 60}] // Max number to be confirmed
+    },
+    errorMessage: settingsErrorMessages.questionTimeLimit
+  },
+  'loadingTimeLimit': {
+    notEmpty: true,
+    isFloat: {
+      options: [{min: 1, max: 5}] // Max number to be confirmed
+    },
+    errorMessage: settingsErrorMessages.loadingTimeLimit
+  }
+}
+
+module.exports.validate = function (req) {
+  return new Promise(async function (resolve, reject) {
+    let validationError = new ValidationError()
+    try {
+      req.checkBody(settingsValidationSchema)
+      let result = await req.getValidationResult()
```

Comment:
Will be modified in the next commit. How about hdf-validator.js? it's using let as well.